cmd/branchingconfigmanagers/schedule-generator/testdata/schedules/4.1.yaml

```diff
@@ -6,7 +6,7 @@ events:
     date: "2020-05-05T16:00:00Z"
   - name: "end-of-full-support"
     date: "2019-11-16T16:00:00Z"
-  - name: "general-availability"
+  - name: "generally-available"
```

Comment:
@stevekuznetsov was there a good reason for why these names should be different in the source and target schedule files?